src/clj_test_containers/core.clj

```diff
@@ -29,6 +34,24 @@
      :env-vars (.getEnvMap container)
      :host (.getHost container)}))
 
+(defn create-from-docker-file
+  [{:keys [exposed-ports env-vars command docker-file]
+    :or {docker-file "Dockerfile"}}]
```

Comment:
In comparison with the Testcontainers Java API as described [here](https://www.testcontainers.org/features/creating_images/), I am seeing three main differences:

1. Defaulting to "Dockerfile" if no `docker-file` is specified is not a behavior I see in the Java API. I would like the general behavior to be mirrored as closesly as possible, though. 

2. The flag to define automatic deletion is not configurable in the Clojure code. 

3. The possibility to pass build args to the Dockerfile is not given in the Clojure code.

Reply:
I am only focusing on one api for now.
I am based my code on the following https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/images/builder/ImageFromDockerfile.java#L209

If you are ok, please feel free to add more to this existing code.
I probably won't have time to fix this for now.

Reply:
I think some of the code you mentioned are not reflected in the current state of the code.
e.g.
I can't find the `withFileFromString` in the Java code. Or if I missed something?

```java
public GenericContainer dslContainer = new GenericContainer(
    new ImageFromDockerfile()
            .withFileFromString("folder/someFile.txt", "hello")
            .withFileFromClasspath("test.txt", "mappable-resource/test-resource.txt")
            .withFileFromClasspath("Dockerfile", "mappable-dockerfile/Dockerfile"
```

Reply:
The `withFileFromString` method is located in the `StringsTrait` interface, which the ImageFromDockerfile class implements: https://github.com/testcontainers/testcontainers-java/blob/f113979ad5207464490cc4f19f488bb27795802c/core/src/main/java/org/testcontainers/images/builder/traits/StringsTrait.java

But it is also okay to me to implement parts of the API for now, I can add the rest to it.